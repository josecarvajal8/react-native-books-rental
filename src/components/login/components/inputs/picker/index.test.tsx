import React from 'react';
import renderer from 'react-test-renderer';
import PickerField from '.';
const mockProps = {
    setFieldValue: () => null,
    fieldId: 'test',
    label: 'test_field',
    value: '',
    options: []
}
test('Render pickerField component correctly', () => {
    const pickerFieldCmp = renderer.create(<PickerField {...mockProps} />).toJSON();
    expect(pickerFieldCmp).toMatchSnapshot();
});