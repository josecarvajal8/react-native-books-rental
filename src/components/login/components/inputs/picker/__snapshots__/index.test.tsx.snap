// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render pickerField component correctly 1`] = `
<View
  forwardedRef={[Function]}
  style={
    Object {
      "borderColor": "#626565",
      "borderRadius": 108.69565217391305,
      "borderWidth": 3.6231884057971016,
      "height": 74.44196428571429,
      "width": 543.4782608695652,
    }
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "marginLeft": 18.115942028985508,
        "marginTop": 14.888392857142858,
        "opacity": 1,
      }
    }
  >
    <Text>
      test_field
    </Text>
  </View>
  <View
    style={
      Object {
        "marginBottom": 14.888392857142858,
        "marginTop": 14.888392857142858,
      }
    }
  />
</View>
`;
